testcases/Crashtool.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+# IBM_PROLOG_BEGIN_TAG
+# This is an automatically generated prolog.
+#
+# $Source: op-test-framework/testcases/Crashtool.py $
+#
+# OpenPOWER Automated Test Project
+#
+# Contributors Listed Below - COPYRIGHT 2022
+# [+] International Business Machines Corp.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+# IBM_PROLOG_END_TAG
+
+import unittest
+import os
+import OpTestConfiguration
+import OpTestLogger
+
+from common.OpTestSystem import OpSystemState
+
+log = OpTestLogger.optest_logger_glob.get_logger(__name__)
+
+class CrashToolInteractiveTest(unittest.TestCase):
+    """
+    Detects and validates the latest crash dump (vmcore) under /var/crash/
+
+    Identifies appropriate debug symbols (vmlinux) based on the OS (e.g., RHEL or SLES)
+
+    Launches crash vmlinux vmcore in interactive mode on the console
+
+    Executes a sequence of commands (log, bt, ps, etc.) in a single session
+
+    Verifies command output and exits cleanly from the crash shell
+    """
+    def setUp(self):
+        conf = OpTestConfiguration.conf
+        self.cv_SYSTEM = conf.system()
+        self.cv_HOST = conf.host()
+        self.console = self.cv_SYSTEM.console.get_console()
+        self.cv_SYSTEM.goto_state(OpSystemState.OS)
+        self.commands = ["log", "bt", "ps", "runq", "kmem -i", "kmem -o", "kmem -h", "vm"]
```

Comment:
Can you also capture o/p of following commands?
- sys
- mach
- bt -a
- kmem -s
- mod